packages/react-router-fs-routes/index.ts

```diff
@@ -43,3 +44,13 @@ export async function flatRoutes(
 
   return routeManifestToRouteConfig(routes);
 }
+
+/**
+ * Creates route config from the file system using a convention that matches
+ * [Remix v1's route file
+ * naming](https://remix.run/docs/en/v1/file-conventions/routes-files), for use
+ * within `routes.ts`.
+ */
+export async function foldersRoutes(): Promise<RouteConfigEntry[]> {
```

Comment:
Could you rename this to `nestedRoutes`?

Also, similar to the `flatRoutes` function above, could you make this accept a "rootDirectory" option, and use `getAppDirectory` internally to resolve the app directory from the user's config?